docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -44,6 +43,16 @@ The Einstein for Developers status icon in the bottom panel of the Visual Studio
 
 You can disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
 
+### Use Einstein for Developers in Code Builder
+
+Einstein for Developers is disabled by default in new Code Builder environments. Telemetry is used to audit enablement of Salesforce Generative AI functionality and must be on when you enable the extension:
+
+1. Go to **Settings** and search for `salesforcedx-vscode-core.telemetry.enabled` to confirm that this Salesforce telemetry setting is on in your workspace.
+2. If the setting isn't on, select the checkbox to enable telemetry, and run **Developer: Reload Window**. 
+3. Click the Einstein icon in the status bar and enable the extension.
+
+Salesforce telemetry is on by default and can be turned off once the extension is enabled.
+
```

Comment:
(for us to discuss): We'll want to add a line here to say CLI telemetry must not be disabled either and point to the docs: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_telemetry.htm